src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -278,7 +278,13 @@ public Build getLastBuild() {
     public <T extends Parameter> T addParameter(Class<T> type) {
         ensureConfigPage();
 
-        check(find(by.xpath("//input[@name='parameterized']")));
```

Comment:
> Do you consider that cleanup a requirement?

Not really, I just feel really sorry for bees that such changes can take days to get integrated and someone have to pick the ball and do the cleanup once we see it fail/succeed. Is there any problem setting up a PR builder, or just nobody did that?